src/System.Collections/src/System/Collections/Generic/Queue.cs

```diff
@@ -204,16 +204,14 @@ public void Enqueue(T item)
         {
             if (_size == _array.Length)
             {
-                int newcapacity = (int)((long)_array.Length * (long)GrowFactor / 100);
-                if (newcapacity < _array.Length + MinimumGrow)
-                {
-                    newcapacity = _array.Length + MinimumGrow;
-                }
-                SetCapacity(newcapacity);
+                Grow();
```

Comment:
Ok. In that case please revert it; if it's beneficial to separate it out, that can be done separately.